microsoft-edge/webview2/reference/javascript/hostobjectsyncproxy.yml

```diff
@@ -135,12 +135,12 @@ methods:
       content: 'setLocalProperty(propertyName: string, propertyValue: any): any;'
       parameters:
         - id: propertyName
-          description: Name of the property to get the value of.
+          description: Name of the property of which to set the value.
           type: string
         - id: propertyValue
           description: Value to set the property to.
           type: any
       return:
         type: any
         description: The value of the property after it is set.
-extends: <a href="https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_es5_d_.callablefunction.html">CallableFunction</a>
+extends: <a href="https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/javascript/hostobjectsyncproxy">CallableFunction</a>

```

Comment:
```suggestion
extends: <a href="https://learn.microsoft.com/microsoft-edge/webview2/reference/javascript/hostobjectsyncproxy">CallableFunction</a>
```

Reply:
in Markdown tagging, LMC links would omit prefix:
/microsoft-edge/webview2/reference/javascript/hostobjectsyncproxy
not sure if applies - build report will indicate

Reply:
you might be still troubleshooting link.  The link text is "**CallableFunction**", but the new link destination 
https://learn.microsoft.com/microsoft-edge/webview2/reference/javascript/hostobjectsyncproxy
has a contradictory title: "**HostObjectSyncProxy class**"

Reply:
closing Outdated comment.  Dave is researching where CallableFunciton is defined.